examples/Main.elm

```diff
@@ -26,38 +25,31 @@ main =
         sweetPollInit =
             SweetPoll.init config
     in
-    Html.program
+    Browser.element
         { init =
-            ( { data = Nothing
-              , error = Nothing
-              , sweetPoll = sweetPollInit |> Tuple.first
-              }
-            , sweetPollInit |> Tuple.second
-            )
+            \_ ->
+                ( { data = Nothing
+                  , error = Nothing
+                  , sweetPoll = sweetPollInit |> Tuple.first
+                  }
+                , sweetPollInit |> Tuple.second
+                )
         , update =
             \action model ->
                 case SweetPoll.update action model.sweetPoll of
                     { sweetPollModel, newData, error, cmd } ->
                         ( { sweetPoll = sweetPollModel
                           , data = newData
-                          , error = error |> Maybe.map toString
+                          , error = error |> Maybe.map Debug.toString
                           }
                         , cmd
                         )
         , view =
             \model ->
                 Html.div []
-                    [ Html.text ("new data: " ++ toString model.data)
+                    [ Html.text ("new data: " ++ Debug.toString model.data)
                     , Html.hr [] []
-                    , Html.text ("error: " ++ toString model.error)
+                    , Html.text ("error: " ++ Debug.toString model.error)
                     ]
-                    |> BeautifulExample.view
```

Comment:
I didn't know about `avh4/elm-beautiful-example` before. Would you want to reinstate this code were it to be upgraded to 0.19 too?

Reply:
Right, I see that you've already answered this in the comments. Sorry, I went straight to the diff.